src/components/Settings/SettingsMain/index.tsx

```diff
@@ -55,6 +56,9 @@ const messages = defineMessages({
   validationApplicationUrlTrailingSlash: 'URL must not end in a trailing slash',
   partialRequestsEnabled: 'Allow Partial Series Requests',
   locale: 'Display Language',
+  discovercustomization: 'Discover Customization',
+  discovercustomizationDescription:
+    'Customize how your users see the discover page.',
```

Comment:
```suggestion
    'Add or remove sliders on the Discover page.',
```
Otherwise the description just restates the heading 😄

Reply:
True. Will change it!